client/components/ChannelWrapper.vue

```diff
@@ -22,7 +22,7 @@
 		:data-name="channel.name"
 		:data-type="channel.type"
 		:aria-controls="'#chan-' + channel.id"
-		:aria-selected="active"
+		:aria-selected="true"
```

Comment:
Drive-by fix here. `active` is not a valid value. It's either `true`, `false`, or undefined (blank)

Reply:
NACK, active is a prop, passed via the element by vue.

Reply:
Ah right. I'm new to Vue and I missed that this was a dynamic attribute. I'll revert this change shortly.